modules/cnv-deleting-custom-resource.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/uninstalling-container-native-virtualization.adoc
+
+[id="cnv-deleting-kubevirt-hyperconverged-custom-resource_{context}"]
+= Deleting the KubeVirt HyperConverged custom resource
+
+To uninstall {ProductName}, you must delete the custom resource that you created
+during deployment.
+
+.Prerequisites
+
+* An active *KubeVirt HyperConverged Cluster Operator Deployment* custom resource
+
+.Procedure
+
+. Ensure that you are in the `kubevirt-hyperconverged` project. If not, select
+it from the *Project* list in the {product-title} web console.
+
+. Navigate to the *Catalog* -> *Installed Operators* page.
+
+. Click *KubeVirt HyperConverged Cluster Operator*.
+
+. Click the *KubeVirt HyperConverged Cluster Operator Deployment* tab.
+
+. Click the Options menu in the row containing the *kubevirt-hyperconverged*
+custom resource. In the expanded menu, click *Delete Hyper Converged*.
+
+. Navigate to the *Workloads* -> *Pods* page to verify that only the Operator
+Pods are running.
+
+. Open a terminal window. Clean up the remaining KubeVirt resources by running
+the following command:
++
+----
+$ oc delete apiservices v1alpha3.subresources.kubevirt.io -n kubevirt-hyperconverged
+----
++
+[NOTE]
+====
+There is an unresolved bug where some KubeVirt resources are not properly
```

Comment:
@ousleyp Consider to add a reference to the bug you mention, please, like we do it in the Known issues section. And I would add that bug in the known issues, too.

Reply:
> @ousleyp Consider to add a reference to the bug you mention, please, like we do it in the Known issues section. 

We don't usually add BZ references in the content, but In this case, I think we can add the specific bug. I'll update the note to be a little more descriptive, too.

> And I would add that bug in the known issues, too.

It's already on my to do list :) https://jira.coreos.com/browse/CNV-1725

Reply:
I have never seen a BZ (workaround) added in the content itself, but if we say that it's a BZ, a reference to it will be reasonable, I believe. Thanks, Pan.

Reply:
I think we decided on this wording in Slack: "Because some KubeVirt resources are currently improperly retained, you must manually remove them. These resources will be removed automatically after \<bug\> is resolved."